data/philly-bike/indego-stations-2022-04-01.csv

```diff
@@ -0,0 +1,190 @@
+Station_ID,Station_Name,Day of Go_live_date,Status
```

Comment:
what is this file? Is this file needed?

Reply:
yes this file is needed because the bike data for philly doesn't have names so I go through this file to add names to the stations

Reply:
You should probably add a README to the data directory (`data/README.md`) explaining this and why the bike data is kept for philly.
The data/README.md file can also detail how data is handled overall vs treated differently for the different areas.